src/panels/MinimalPanel.vue

```diff
@@ -10,42 +10,51 @@
-            <span class="card-title-inline"><slot name="_header"></slot></span>
-            <span v-show="showDownSwitch" aria-hidden="true"
-                  class="minimal-button glyphicon glyphicon-menu-down minimal-menu-down"></span>
-          </slot>
-        </span>
-        <div :class="['button-wrapper', { 'button-wrapper-expanded': localExpanded, 'button-wrapper-visible': onHeaderHover }]">
+        <transition name="title">
+          <span v-show="!isHeaderAtBottom" ref="headerWrapper"
+                :class="['card-title', 'card-title-transparent', { 'card-title-opaque': onHeaderHover, 'ellipses': !hasHeaderBool }]">
+            <slot name="header">
+              <span class="card-title-inline"><slot name="_header"></slot></span>
+              <span v-show="showDownSwitch" aria-hidden="true"
+                    class="minimal-button glyphicon glyphicon-menu-down minimal-menu-down"></span>
+            </slot>
+          </span>
+        </transition>
+        <div :class="['button-wrapper', { 'button-wrapper-expanded': isHeaderAtBottom, 'button-wrapper-visible': onHeaderHover }]">
           <slot name="button">
             <button v-show="!noCloseBool" class="minimal-button" type="button" @click.stop="close()">
               <span class="glyphicon glyphicon-remove minimal-close-button" aria-hidden="true"></span>
             </button>
             <button v-show="popupUrl !== null" class="minimal-button" type="button" @click.stop="openPopup()">
               <span class="glyphicon glyphicon-new-window minimal-popup-button" aria-hidden="true"></span>
             </button>
-            <button v-show="localExpanded && isExpandableCard && bottomSwitchBool" class="minimal-button" type="button"
-                    @click.prevent.stop="collapseThenScrollIntoViewIfNeeded()">
+            <button v-show="isHeaderAtBottom && isExpandableCard && bottomSwitchBool" class="minimal-button" type="button">
               <span class="glyphicon glyphicon-menu-up minimal-menu-up" aria-hidden="true"></span>
             </button>
           </slot>
         </div>
       </div>
-      <template v-if="preloadBool || isCached">
-        <div class="card-collapse"
-             ref="panel"
-             v-show="localExpanded"
-        >
+      <transition @before-enter="beforeExpandMinimal"
+                  @enter="duringExpand"
+                  @leave="duringCollapse"
+                  @after-leave="afterCollapseMinimal"
+                  v-if="preloadBool || isCached"
+      >
+        <div v-show="localExpanded"
+             class="card-collapse"
+             ref="panel">
           <div class="card-body">
             <slot></slot>
-            <retriever v-if="hasSrc" ref="retriever" :src="src" :fragment="fragment"/>
+            <retriever v-if="hasSrc"
+                       ref="retriever"
+                       :src="src"
+                       :fragment="fragment"
+                       @src-loaded="setMaxHeight"/>
```

Comment:
`setMaxHeight` fires once the content has loaded, the rest of the code centers around this.
( the max height needs to be known for the animation to work )